content/enterprise_influxdb/v1.9/administration/manage/security/introduction-to-auth.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Introduction to authentication and authorization
+description: >
+  Learn the basics of managing authentication and authorization in InfluxDB Enterprise.
+menu:
+  enterprise_influxdb_1_9:
+    name: Introduction to authentication and authorization
+    parent: Manage users and permissions
+weight: 30
+related:
+  - /enterprise_influxdb/v1.9/guides/fine-grained-authorization/
+  - /{{< latest "chronograf" >}}/administration/managing-influxdb-users/
+---
+
+To secure and manage access to an InfluxDB Enterprise cluster, consider the following two aspects:
+- *authentication* (verifying a user's identity)
+- *authorization* (verifying what the user has access to)
+
+{{% enterprise-warning-authn-b4-authz %}}
+
+## Permissions in InfluxDB Enterprise
+
+InfluxDB Enterprise has an [expanded set of 16 permissions](/enterprise_influxdb/v1.9/administration/manage/security/permissions/#permissions).
+These permissions allow for
+controlling read and write access to data for all databases at once,
+as well as permitting certain cluster-management actions like creating or deleting resources.
+
+InfluxB 1.x OSS only supports database-level privileges: `READ` and `WRITE`.
+A third permission, `ALL`, grants admin privileges.
+These can only be _can only be granted by using InfluxQL_.
+
+## Manage user authorization
+
+There are three ways to manage authorizations in InfluxDB Enterprise:
+
+- using [InfluxQL](#manage-read-and-write-privileges-with-influxql)
+  {{% note %}}
+InfluxQL can can only grant `READ`, `WRITE`, and `ALL PRIVILEGES` privileges.
+To use the full set of InfluxDB Enterprise [permissions](/enterprise_influxdb/v1.9/administration/manage/security/permissions/),
+use [Chronograf](#manage-specific-privileges-with-chronograf)
+or the [Meta API (recommended)](#influxdb-enterprise-meta-api).
```

Comment:
Feels like duplicate info from above...can we combine to streamline? May be something like:

- To use full set of InfluxDB Enterprise permissions, use one of the following:
   - (**Recommended**)...
   - Chronograf...
- To grant `READ`, `WRITE`, and `ALL PRIVILEGES` privileges only, use [InfluxQL](#manage-read-and-write-privileges-with-influxql).

Reply:
Both of the following are important and true:

- InfluxQL can can only grant `READ`, `WRITE`, and `ALL PRIVILEGES` privileges.
- These permissions can only be granted by using InfluxQL.